### PR TITLE
Data Documentation for load_espn_qbr

### PR DESCRIPTION
**Pull request by pranavrajaram**
Hey guys,

I thought I'd try and give back to the nflreadr because of how helpful it is, so I started to work on one of the "good first issues" that were listed. To start,  I created a data dictionary for the "load_espn_qbr" package which is what this pull request is for. Everything seems to work fine on my end in R (if I load in my forked github using devtools), but I'm not sure on how to test if my updates are also shown in the pkgdown website. 

Please let me know if I did anything wrong, and any feedback would be much appreciated. Thanks for all you do for the community!

### CODE REVIEW

**Comment by tanho63** [comment]
Awesome! You can check locally to see if the pkgdown will build by running `devtools::build_site()` (or pkgdown's equivalent).  I'll have a quick peek here around lunch

File: R/load_nflverse.R [hunk 1]
```diff
@@ -324,6 +324,8 @@ load_injuries <- function(seasons = most_recent_season(),
 #' @return a tibble of season-level injury report data.
 #'
 #' @seealso <https://github.com/nflverse/espnscrapeR-data>
+#' @seealso [`dictionary_espn_qbr`] for the data dictionary
+#' @seealso <https://nflreadr.nflverse.com/articles/dictionary_espn_qbrhtml> for a web version of the dictionary
```
Comment (tanho63) [inline R/load_nflverse.R:328]: This link is probably borked (needs . between qbr and html)

**Comment by tanho63** [comment]
Can you also run `devtools::document()` twice?

**Review by tanho63 — CHANGES_REQUESTED** [review]
See typo note + please redocument with `devtools::document()` (usually twice), then commit/push those here and the checks should pass

**Comment by pranavrajaram** [comment]
thanks for getting back! might be a dumb question, but where should I run devtools::document()?

**Comment by tanho63** [comment]
Just in your console, it'll operate over the whole package

**Review by tanho63 — APPROVED** [review]
LGTM

**Pull request merged**